policy/README.md

```diff
@@ -248,6 +248,7 @@ An individual `Rule` object is defined by the following fields:
 | `days`             | day[]                       | Optional   | Days `["sun", "mon", "tue", "wed", "thu", "fri", "sat"]` when the rule is in effect (default all) |
 | `messages`         | `{ String:String }`         | Optional   | Message to rider user, if desired, in various languages, keyed by language tag (see [Messages](#messages)) |
 | `value_url`        | URL                         | Optional   | URL to an API endpoint that can provide dynamic information for the measured value (see [Value URL](#value-url)) |
+| `stops`      | UUID[]                      | Optional   | List of [Stop](/general-information.md#stops) UUIDs that this Rule applies to. If this field is used then the `geographies` field must be used to identify the locations.  |
```

Comment:
We talked at the working group meeting today about the potential for errors with having stops and geographies both point to geographical entities. We also discussed if there would be other geographical entities and if that would lead to a proliferation of fields. One proposed solutions is to replace `stops` and `geographies` with a field called `places` which is a map where the keys indicate another API and the values are a list of UUIDS. So for example:
```
{ 'geographies': [UUIDS], 'stops': [UUIDS]' }
```
This would be breaking if we replaced `geographies`. We could add this field instead of `stops` and incorporate 'stops', and maybe 'geographies', as allowed keys here. If we go this route we will want to update the description of the geographies field to indicate that this change is happening.

Reply:
+1 to adding top-level `places` now instead of `stops`, and duplicating the data from `geographies`. This seems like an OK non-breaking approach which can be further cleaned up in 2.0 timeframe.

Reply:
I think that for this release we should just pull out this `stops` option, work out the best solution to the larger idea in the next release, and not have to duplicate geographies data now.  

For naming, `places` is now used within Stops, eg, a place to park a vehicle within a Stop area. Naming this places I think would be confusing.  Maybe `area`?